geometric_kernels/spaces/base.py

```diff
@@ -39,14 +39,14 @@ class DiscreteSpectrumSpace(Space):
     @abc.abstractmethod
     def get_eigenfunctions(self, num: int) -> Eigenfunctions:
         """
-        First `num` eigenfunctions of the Laplace-Beltrami operator
+        Eigenfunctions of the first `num` levels of the Laplace-Beltrami operator.
```

Comment:
"Eigenfunctions of the first `num` levels of the Laplace-Beltrami operator." -> "Eigenfunctions of the Laplace-Beltrami operator corresponding to the first `num` levels."